nengo_viz/static/viz_scale.js

```diff
@@ -80,5 +74,6 @@ VIZ.scale.redraw_size = function(scale_x, scale_y) {
 		var w = VIZ.Component.components[i].width;
 		var h = VIZ.Component.components[i].height;
 		VIZ.Component.components[i].on_resize(w * scale_x, h * scale_y);
+		VIZ.Component.components[i].save_layout();
```

Comment:
The rest of the code avoids calling save_layout() inside a utility function like redraw_size.  Instead, it's always calling save_layout() from within the event handler (or at least, that was the intent -- if there are places I missed, let me know).  That way we avoid having a whole bunch of save_layout() calls happening as a side effect of a single action (which just overwhelms the communication between the client and server).  So if I put a save_layout() call inside VIZ.scale.redraw_size, then that means that any event that ends up calling redraw_size and doing something else is in danger of doing redundant saving.